ansible_collections/arista/avd/docs/containers/overview.md

```diff
@@ -0,0 +1,92 @@
+<!--
+  ~ Copyright (c) 2023 Arista Networks, Inc.
+  ~ Use of this source code is governed by the Apache License 2.0
+  ~ that can be found in the LICENSE file.
+  -->
+
+# AVD Dev Containers Overview
+
+> WARNING: AVD Dev Containers are currently in the early preview phase and not officially supported for any production cases.
+> They can still be very useful for development and testing. Please report any issues and optimization suggestions.
```

Comment:
not sure if this renders well on mkdocs

we use admonittion syntax `!!!` (and make github sad, yes)

Reply:
changed in my commit. Resolving this.